libnopegl/src/text.c

```diff
@@ -140,7 +137,7 @@ int ngli_text_set_string(struct text *s, const char *str)
         }
 
         const struct char_info chr = {
-            .atlas_id = ngli_drawutils_get_atlas_id(str[i]),
+            .atlas_id = s->ctx->char_map[(uint8_t)str[i]],
```

Comment:
Why casting to uint8_t here ? Would adding an assert makes more sense ?

Reply:
To prevent this warning: `warning: array subscript has type ‘char’ [-Wchar-subscripts]`

Reply:
Also would adding an assert or a check on the value make sense ?

Reply:
I'm not sure how it could overflow; `char_map` covers the whole 8-bit range and `str[i]` is 8-bit (just, signed).

Reply:
Ok.